regression-tests.api/test_Cache.py

```diff
@@ -0,0 +1,22 @@
+from test_helper import ApiTestCase, is_auth, is_recursor
+
+
+class Servers(ApiTestCase):
```

Comment:
Right now, during tests, the cache is always empty, and I don't want to add cache-prefilling now. When cache-snooping is in, we can add proper tests for that; for now I'll go with believing the server it did whatever it thinks it needs doing.